install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1078,6 +1078,8 @@ gke:
 healthChecking: true
 # -- TCP port for the agent health API. This is not the port for cilium-health.
 healthPort: 9879
+# -- Number of ICMP requests sent for each health check before marking a node or endpoint unreachable.
+healthCheckICMPProbesCount: 1
```

Comment:
non-blocking suggestion: make this 2 or 3?

Reply:
I was initially thinking to keep the original behavior (a single probe) but actually a higher default seems sane. Changed to 3 :+1: